spyder/plugins/editor/confpage.py

```diff
@@ -39,6 +39,12 @@ def setup_page(self):
                 'show_class_func_dropdown')
         showindentguides_box = newcb(_("Show Indent Guides"),
                                      'indent_guides')
+        show_hover_hints_box = newcb(
+            _("Show hover hints"),
+            'show_hover_hints',
+            tip=_("If this option is enabled, hovering the mouse \n"
+                  "pointer over an object name will display a hint of\n"
+                  "that object's definition (if resolved)."))
```

Comment:
I think we should move this option to `lsp-server` and use it through CONF in CodeEditor.

It's just odd to have almost all options in "Completion and linting", but not this one.